cmake/README.md

```diff
@@ -50,14 +50,7 @@ Here you can find complete list of Xournal++ CMake flags (sorted by categories).
 
 | Variable name        | Default | Description
 | -------------------- | ------- | -----------
-| `ENABLE_CPPUNIT`     | OFF     | Build CppUnit test instead of xournalpp application
-
-
-## `TEST` – optional features of CppUnit tests
-
-| Variable name      | Default | Description
-| ------------------ | ------- | -----------
-| `TEST_CHECK_SPEED` | OFF     | Show speed benchmarks for some tests
+| `ENABLE_GTEST`       | OFF     | Download and build GoogleTest (if not previously done) and build tests instead of xournalpp application
```

Comment:
Note at first glance the alignment looks of but the new line needs to be compared to line 52.